kernelci/cli/user.py

```diff
@@ -44,10 +46,9 @@ def whoami(config, api, indent, secrets):
 @catch_http_error
 def find(attributes, config, api, indent):
     """Find user profiles with arbitrary attributes"""
-    configs = kernelci.config.load(config)
-    api_config = configs['api'][api]
-    api = kernelci.api.get_api(api_config)
-    users = api.get_users(split_attributes(attributes))
+    api = get_api(config, api)
+    attributes = split_attributes(attributes)
+    users = api.get_users(attributes)
```

Comment:
We should also use `get_pagination` here for `get_users`.

Reply:
Yes, I'll let you deal with this as a follow-up I think.  This PR is just about completing the `kci node` command tool.